build.gradle

```diff
@@ -65,7 +65,7 @@ publishing {
 
 dependencies {
     implementation "org.eclipse.lsp4j:org.eclipse.lsp4j:0.9.0"
-    implementation "software.amazon.smithy:smithy-model:1.1.0"
+    implementation "software.amazon.smithy:smithy-model:1.5.1"
```

Comment:
Hmmm.... You don't have to do this, but we should probably make this a range dependency